control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -467,7 +467,8 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
                     Ok(node) if node.state.as_ref().map(|n| n.status).unwrap_or(NodeStatus::Unknown) != NodeStatus::Online || cordon_check(node.spec.as_ref()) => {
                         Ok(None)
                     },
-                    Ok(_) => Ok(Some(node_id.as_str()))
+                    Ok(_) if volume.spec.num_replicas > 1 => Ok(Some(node_id.as_str())),
```

Comment:
This could do with a comment explaining what's happening here.

Reply:
Done